app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/landing/LandingViewModel.kt

```diff
@@ -191,8 +194,12 @@ class LandingViewModel @Inject constructor(
     }
 
     private fun handleCreateAccountClicked() {
-        // TODO PM-9401: ADD FEATURE FLAG email-verification and navigation to StartRegistration
-        sendEvent(LandingEvent.NavigateToCreateAccount)
+        val navigationEvent = if (featureFlagManager.getFeatureFlag(key = FlagKey.OnboardingFlow)) {
```

Comment:
👍

Reply:
This needs further confirmation, other platforms are using `'email-verification'` feature flag for the new account creation flow.

Reply:
@andrebispo5 Are you saying there are 2 feature flags for 1 feature?

Reply:
source: [clients routing](https://github.com/bitwarden/clients/blob/4d088f38565a9c830ac09cbaf59f9b5e264be3c7/apps/desktop/src/app/app-routing.module.ts#L122)

Reply:
@andrebispo5 Micah does not want this stuff enabled till the whole feature is complete on the apps, but the flag will be turned on Sept. 3 for safety we are going to leave this for now, so in the event that flag is turned on and the new flow is not ready the web can still go live. 

I am thinking of making it a 
```pseudo
if (OnBoarding && EmailVerification) doTheNewThing else keepItOldSkool